spec/features/editing_content/edit_a_document_spec.rb

```diff
@@ -10,6 +10,8 @@
     and_the_content_is_available_for_preview
     and_the_content_needs_review
     and_i_see_the_content_is_in_draft_state
+    and_i_see_i_was_the_last_user_to_edit_the_document
+    and_on_the_index_page_i_see_the_document_was_last_edited_by_me
```

Comment:
This last step is doing a lot - could this check be done in the index page spec instead?

Reply:
The index page specs seem to be focused on filtering, which is why I chose not to add this test to one of those files.  This seemed to be the best place to put it given that it's related to editing a document, though I agree it's perhaps a little shoehorned in.  Would it help if I split it into visiting the index page and seeing the last editor as separate steps?

Reply:
I think it would be better to create a quick 'index_page' spec to check content that appears on the index page - we're not checking most of it at the moment. Line 13 already checks the edit feature works, so line 14 is only checking that we're showing this information on the index page and is therefore more related to a spec about the index page specifically.

Reply:
Added here: https://github.com/alphagov/content-publisher/pull/336/commits/e3792f23ed6238a0be1ec103c7e136c34687945e